From ea8ce23b5e0d20cd56f8718fb0485049e0ce53ca Mon Sep 17 00:00:00 2001
From: avishad verma <avishad.verma@infineon.com>
Date: Tue, 10 Jan 2023 03:41:20 -0600
Subject: [PATCH 202/208] brcmfmac: SR in not getting enabled in H1

It has been found in case the Chip ID is present we are not populating
the PMU capabilities. Due to which the SR is not getting enabled.
While enabling the SR we check for PMU revision because chips with
older PMU revision do not support SR.



---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c
index d0f39099096f..8de88b1829ad 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/chip.c
@@ -1298,9 +1298,6 @@ static int brcmf_chip_setup(struct brcmf_chip_priv *chip)
 	u32 val;
 	int ret = 0;
 
-	if (chip->pub.ccsec)
-		return 0;
-
 	pub = &chip->pub;
 	cc = list_first_entry(&chip->cores, struct brcmf_core_priv, list);
 	base = cc->pub.base;
-- 
2.17.1

